### ISSUE TITLE
Pass accessibility tests

### ISSUE THREAD

**Issue by jackiejohnston** [formvalidation/support#823]
I was running some accessibility tests on a page with a form with your form validation, and it fails because of a hidden button that is added to the form. (Error: "A button is empty or has no value text.") 

```
<button type="submit" class="fv-hidden-submit" style="display: none; width: 0px; height: 0px;"></button>
```

A quick fix you should probably add to your code on `formValidation.js` line 242 add button text:

```
            // When pressing Enter on any field in the form, the first submit button will do its job.
            // The form then will be submitted.
            // I create a first hidden submit button
            this.$hiddenButton = $('<button/>')
                                    .attr('type', 'submit')
                                    .prependTo(this.$form)
                                    .addClass('fv-hidden-submit')
                                    .css({ display: 'none', width: 0, height: 0 });
                                    // add this line:
                                    .text('Submit');
```
